network/src/peer/mod.rs

```diff
@@ -447,16 +402,14 @@ where
                 }
                 ReadError::IoError(_) => {
                     // IoErrors are mostly unrecoverable so just close the connection.
-                    self.close_connection(DisconnectReason::ConnectionLost);
+                    self.shutdown(DisconnectReason::ConnectionLost);
```

Comment:
So, do we no longer close the connection?  Oh, nevermind I see when it comes around on the shutdown it closes it.